user-profile-card.js

```diff
@@ -2,14 +2,19 @@ import '@brightspace-ui/core/components/colors/colors.js';
 import '@brightspace-ui/core/components/icons/icon.js';
 import { bodySmallStyles, bodyStandardStyles, heading2Styles, labelStyles } from '@brightspace-ui/core/components/typography/styles.js';
 import { css, html, LitElement } from 'lit-element/lit-element.js';
+import {classMap} from 'lit-html/directives/class-map.js';
+import { inputStyles } from '@brightspace-ui/core/components/inputs/input-styles.js';
 import { LocalizeMixin } from '@brightspace-ui/core/mixins/localize-mixin.js';
 
 class UserProfileCard extends LocalizeMixin(LitElement) {
 
 	static get properties() {
 		return {
+			_isEditing : { type : Boolean },
+			editable: {type: Boolean},
 			online: { type: Boolean },
-			userAttributes: { type: Array, attribute: 'user-attributes', reflect: true }
+			userAttributes: { type: Array, attribute: 'user-attributes', reflect: true },
+			tagline : { type : String, reflect: true }
```

Comment:
Nit: Keep things consistent with the spacing on the objects - I see three different variations in the properties you added
```suggestion
			tagline: { type: String, reflect: true }
```